contributing/chestertons-fence.md

```diff
@@ -0,0 +1,79 @@
+# Chesterton's Fence
+
+Hello :wave:
+You're probably here becomes someone directed you to this post in response to you suggesting an idea about how Exercism could be improved.
+Before our team invest time into exploring your idea, they're hoping you'll read this and consider whether your idea might of being thought of before, what pitfalls it might contain.
```

Comment:
```suggestion
Before our team invests time into exploring your idea, they're hoping you'll read this and consider whether your idea might have been thought of before, what pitfalls it might contain.
```